### ISSUE TITLE
Ability to define custom user fields

### ISSUE THREAD

**Issue by molul** [sjarvela/kloudspeaker#146]
Hi. I love kloudspeaker simplicity, but I'd like to have a couple of extra fields for users info (like country, phone, etc). Are you planning on adding custom fields? :)

**Comment by sjarvela** [comment]
You mean into user data?

At the moment, kloudspeaker stores only data it needs, all other would not be relevant to all users. This is why I won't add any fields there by default, it would have to be some custom definition that each user (installation) specifies.

I'll modify this issue to progress with this aspect.

**Comment by molul** [comment]
Yes, sorry for not being more specific. I meant having extra fields for the user profile. Currently, when you create an user account, you can set username, email, password, etc. I'd like to be able to add phone, company, country... 

For instance, this would be useful for me to create users reports classified by country.
